### DISCUSSION TITLE
How to enroll in Github Octernship Program

### DISCUSSION THREAD

**Discussion by avinashrajputt** [community/community#139183] [GitHub Education]
### Select Topic Area

General

### Body

How to get into Octernship Program

**Comment by Chanduporalla** [comment]
To enroll in the GitHub Octernship program, follow these steps:

    Visit the GitHub Education Website: Go to the [GitHub Octernship page](https://education.github.com/students/octernships) for official details.

    Eligibility: Ensure you are eligible. GitHub Octernship is open to students who:
        Are enrolled in a university or college.
        Have a GitHub Student Developer Pack (which is a requirement for many of GitHub’s student programs).

    Get the GitHub Student Developer Pack:
        If you don’t have it yet, apply for the [GitHub Student Developer Pack](https://education.github.com/pack) by verifying your student status.

    Check Available Octernships:
        On the GitHub Octernship page, view available internships or projects that match your skills and interests.

    Apply for Octernships:
        Applications for specific Octernship opportunities will have different processes, but typically you’ll need to submit a project or complete an assignment based on the internship you’re applying for. The Octernship listing should provide details on the submission process.

    Follow Guidelines: Pay close attention to the requirements, deadlines, and any specific tasks required for the internship you're interested in.

    Stay Updated:
        Follow GitHub Education on [social media](https://twitter.com/GitHubEducation) or subscribe to their newsletter to stay updated on new Octernship opportunities.

**Reply by Shurtu-gal** [reply]
I suppose it is closed for now.
